src/java/edu/slu/tpen/transfer/JsonLDExporter.java

```diff
@@ -79,6 +79,26 @@ public JsonLDExporter(Project proj, User u, String profile) throws SQLException,
                manifestData.put("label",buildNoneLanguageMap(proj.getProjectName()));
                manifestData.put("metadata", getMetadataAsJSON(projID, profile));
               
+               Map<String, Object> services;
+           services = new LinkedHashMap<>();
+               services.put("id","http://t-pen.org/TPEN/login.jsp");
+               services.put("type",buildNoneLanguageMap("ImageService3"));
```

Comment:
From https://iiif.io/api/presentation/3.0/#services this should be `type` and a value for it a string, not language map